### DISCUSSION TITLE
How can I migrate from jlesage/docker-nginx-proxy-manager to the official image?

### DISCUSSION THREAD

**Discussion by strenkml** [NginxProxyManager/nginx-proxy-manager#2369] [Q&A]
I use Unraid and when I setup Nginx Proxy Manager there was not an official app on Unraid, there was only [this one](https://github.com/jlesage/docker-nginx-proxy-manager). I see that there now is an official Unraid app and I would like to switch to using that one.

It does not seem like it would be possible to just point the new app at the folder from the old installation. Is there a way to export/import the config or would it just be better to re-setup the app using the official version?

**Comment by mrfuntime** [comment]
It appears the only thing you are gaining from running the "official" image is IPv6 support.  If you are not running IPv6 on your network, then it's likely not worth your time and energy.
